portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,7 +64,17 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOExc
 
     datastore.put(commentEntity);
 
+    Query query = new Query("Comment");
+    PreparedQuery results = datastore.prepare(query);
+
+    ArrayList<String> comments = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("content");
+      comments.add(comment);
+    }
+
     // Redirect back to the HTML page.
+
```

Comment:
Remove this line (keep the comment with code)

Reply:
Resolving because removed